internal/elasticsearch/transform/transform.go

```diff
@@ -789,18 +829,18 @@ func flattenSync(sync *models.TransformSync) []interface{} {
 		return nil
 	}
 
-	time := make(map[string]interface{})
+	t := make(map[string]interface{})
 
 	if sync.Time.Delay != "" {
-		time["delay"] = sync.Time.Delay
+		t["delay"] = sync.Time.Delay
 	}
 
 	if sync.Time.Field != "" {
-		time["field"] = sync.Time.Field
+		t["field"] = sync.Time.Field
 	}
 
 	s := make(map[string]interface{})
-	s["time"] = []interface{}{time}
+	s["t"] = []interface{}{t}
```

Comment:
```suggestion
	s["time"] = []interface{}{t}
```

The underlying schema isn't changing here.